frontend/pages/events/[id].vue

```diff
@@ -139,21 +140,26 @@ const toggleContact = () => {
   showContact.value = !showContact.value;
 };
 
+
 onMounted(async () => {
   try {
     const eventId = route.params.id;
     const response = await $fetch(
       `${config.public.API_BASE_URL}/api/events/${eventId}`
     );
-    console.log(response);
-
     event.value = response;
-    console.log(event.value);
     loaded.value = true;
   } catch (error) {
-    console.error("Error fetching event data", error);
+    if (error.response && error.response.status === 404) {
+      router.push('/404');
+    } else {
+      console.error('Error fetching event data:', error);
+    }
```

Comment:
Can you please add a toaster message here with a bit specific error description ?

Reply:
The required changes are addressed in the latest commit